app/query_param.go

```diff
@@ -21,7 +23,13 @@ func ParseHistoryQueryParams(c echo.Context, defaultLimit int64) (string, int64,
 		from = time.Now().Unix()
 	}
 
-	return userID, from, limit
+	if err != nil {
+		// If it returns error, it will assume the default behavior(e.g. isBlocked=false).
+		fmt.Println("ERROR:", err)
```

Comment:
Is there a way to use our log engine here (`logrus`)? It could be better to get a structured log later.

Reply:
[Done, captain](https://github.com/topfreegames/mqtt-history/pull/20/commits/60f68bdbd84f4ea1bdac99c5f80de67a75852ce2) 👍🏾